Doppler.reportingApi/Controllers/DetailController.cs

```diff
@@ -0,0 +1,15 @@
+using Doppler.ReportingApi.Models;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Doppler.ReportingApi.Controllers
+{
+    [ApiController]
+    public class DetailController : ControllerBase
```

Comment:
What do you think about using POCO controllers?

```suggestion
    public class DetailController
```

Reply:
Its a good idea :)